core/src/main/java/pl/agh/edu/model/time/Time.java

```diff
@@ -121,4 +122,8 @@ public int getTimeUnitInMinutes() {
 		return timeUnitInMinutes;
 	}
 
+	public MonthDay getCurrentMonthDay() {
```

Comment:
ja bym bez nazwał getMonthDay. (skoro pytamy czas o dzień to raczej będzie to obecny)